fhir-bulkimportexport-webapp/src/main/java/com/ibm/fhir/bulkcommon/BulkDataUtils.java

```diff
@@ -134,9 +137,18 @@ public static void listBuckets(AmazonS3 cosClient) {
         }
     }
 
+    /**
+     * @param resReader - the buffer reader to read FHIR resource from.
+     * @param numOfLinesToSkip - number of lines to skip before read.
+     * @param fhirResources - List holds the FHIR resources.
+     * @return - the number of parsing failures.
+     * @throws Exception
+     */
     private static int getFhirResourceFromBufferReader(BufferedReader resReader, int numOfLinesToSkip, List<Resource> fhirResources) throws Exception {
         int exported = 0;
         int lineRed = 0;
+        int parseFailures = 0;
+
         String resLine = null;
```

Comment:
you can switch back to a while loop

Reply:
en, I would prefer to keep it unchanged ...

Reply:
no worries - it wasn't a requirement - just an observation